docs/scripts/plugin_documentation

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+function plugin_doc {
+	plugin=$1
+	dest=$2
+	product=`echo $plugin | sed -e 's/-.*//'`
+	if [ $product = "foglamp" ]; then
+		org=dianomic
+	else
+		org=fledge-iot
+	fi
+	type=`echo $plugin | sed -e 's/fledge-//' -e 's/foglamp-//' -e 's/-.*//'`
+	name=`echo $plugin | sed -e 's/fledge-//' -e 's/foglamp-//' -e "s/${type}-//"`
+	mkdir -p /tmp/doc.$$
+	(
+		cd /tmp/doc.$$
+		git clone https://github.com/${org}/${plugin}.git
+		if [ "$DOCBRANCH" != "" ]; then
+			cd $plugin
+			git checkout $DOCBRANCH
+		fi
```

Comment:
This lines of code is not used as variable DOCBRANCH is not defined.

Reply:
DOCBRANCH is there to allow you to test documentation changes before the changes get merged to develop.